### DISCUSSION TITLE
串流启动时光标错位

### DISCUSSION THREAD

**Discussion by waw1w1** [Blinue/Magpie#1030] [General]
串流时因为缩放问题控制端的光标位置和实际位置是错位的，这个请问有解决办法吗？

![IMG_20241209_185748](https://github.com/user-attachments/assets/bcf750b0-8ec6-4c1f-8384-1c92ba8f108d)

**Comment by Blinue** [comment]
看看 #141

**Reply by waw1w1** [reply]
感谢大佬
